winterfell/webapp/src/js/claimSelectFormsCtrl.js

```diff
@@ -1,13 +1,23 @@
 var app = angular.module('vetafiApp');
-app.controller('claimSelectFormsCtrl', ['$scope', 'claimService', 'formTemplateService', '$stateParams',
-  function($scope, claimService, formTemplateService, $stateParams) {
+app.controller('claimSelectFormsCtrl', ['$scope', 'claimService', '$stateParams',
+  function($scope, claimService, $stateParams) {
     $scope.claimId = $stateParams.claimId;
-    $scope.forms = formTemplateService;
+    $scope.allForms = [
```

Comment:
It is at the top level because it is just a placeholder that imports the forms via browserify. Because it is the only thing that uses browserify i wanted to separate it out